products/jbrowse-cli/src/base.ts

```diff
@@ -52,6 +121,14 @@ export default abstract class JBrowseCommand extends Command {
     return fsPromises.readFile(location, { encoding: 'utf8' })
   }
 
+  async writeJsonConfig(config: string) {
+    try {
+      fsPromises.writeFile('./config.json', config)
+    } catch {
```

Comment:
might make sense to to catch(e) and directly log this.error(`${e}`) since we will get the permission denied, or other type error. and then, in that case, might make sense to not have an exception handler anyways because i think there is a default one that would have this behavior

Reply:
So your suggestion is to remove exception handler and just call writeFile?

Reply:
Switched this to `` `this.error(${error})` ``.